### ISSUE TITLE
Subgroup of automorphism group hanging(?)

### ISSUE THREAD

**Issue by thofma** [oscar-system/Oscar.jl#1531]
In the following [gist](https://gist.github.com/thofma/14509201ac7178b5580b7f33080a89b5), an abelian group and a list of automorphisms is defined. When trying to construct the corresponding subgroup of the automorphism group, Oscar hangs (ctrl-c yields no stacktrace, probably thanks to arm64 on mac).

```
OSCAR version 0.10.2 - #master, 2e42af0e0 -- 2022-09-09 22:50:32 +0200
  combining:
    AbstractAlgebra.jl   v0.27.4 - #master, 85c36e73b -- 2022-08-27 16:47:43 +0200
    GAP.jl               v0.8.2
    Hecke.jl             v0.14.13
    Nemo.jl              v0.32.2 - #master, 8017f6fd -- 2022-09-01 12:38:17 +0200
    Polymake.jl          v0.8.0
    Singular.jl          v0.12.0
  building on:
    Antic_jll               v0.201.500+0
    Arb_jll                 v200.2300.0+0
    Calcium_jll             v0.401.100+0
    FLINT_jll               v200.900.1+0
    GAP_jll                 v400.1192.2+1
    Singular_jll            v403.100.100+0
    libpolymake_julia_jll   v0.8.1+0
    libsingular_julia_jll   v0.25.0+0
    msolve_jll              v0.4.4+0
    polymake_jll            v400.700.1+1
See `]st -m` for a full list of dependencies.

Julia Version 1.9.0-DEV.1013
Commit 8378d4cf96* (2022-07-20 02:54 UTC)
Platform Info:
  OS: macOS (arm64-apple-darwin20.6.0)
  CPU: 8 × Apple M1
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-14.0.5 (ORCJIT, apple-m1)
  Threads: 1 on 8 virtual cores
```

@fingolfin @ThomasBreuer

**Comment by ThomasBreuer** [comment]
@thofma The construction of the subgroup calls `GAP.Globals.Subgroup( G, gens )`, which wants to test whether the elements in `gens` are elements of `G`; this triggers the computation of an isomorphic permutation group. Roughly speaking, any question about `G` (for example, what the order of `G` is) will be delegated to an isomorphic permutation group.

It would make sense to provide a way to omit these membership checks in situations where one knows that they are unnecessary.
This would help here if one is interested only in the subgroup. Questions about the subgroup will also be delegated to an isomorphic permutation group, but computing with this subgroup may be easier.

**Comment by thofma** [comment]
Thanks for the explanation (I wondered what computation the `sub` had triggered). My subgroups have small size compared to the full automorphism group, so I had hope that computing a permutation presentation might be doable.

Is disabling the membership test something we can do in Oscar or does it require adjustments in GAP?

**Comment by ThomasBreuer** [comment]
I will create a pull request for Oscar.

**Comment by thofma** [comment]
Fixed in #1532. Thanks @ThomasBreuer.
